modules/jfx.incubator.richtext/src/test/java/test/jfx/incubator/scene/control/richtext/support/RTUtil.java

```diff
@@ -75,6 +76,27 @@ public static String getText(RichTextArea control) {
         }
     }
 
+    /**
+     * Extracts plain text from the supplied StyledTextModel, using {@code write(DataFormat.PLAIN_TEXT)} method.
+     *
+     * @param model the model
+     * @return the plain text
+     */
+    public static String getText(StyledTextModel model) {
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        try {
+            try {
```

Comment:
Minor: you don't need two levels of try here. Even better, you can use try with resources instead of adding finally clause.